.travis.yml

```diff
@@ -1,32 +1,24 @@
 language: python
+cache: pip
 services:
   - docker
 env:
   global:
-    - DOCKER_COMPOSE_VERSION: 1.25.5
     - SERVICE_NAME: scorpio-web
```

Comment:
I know this is nitpicky but can we call this $CONTAINER so it's consistent across apps?